packages/Microsoft.Bot.Components.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30907.101
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Bot.Component.CalendarSkill", "CalendarSkill\Microsoft.Bot.Component.CalendarSkill.csproj", "{84AEC10B-B272-4884-B434-35052B866D60}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Bot.Component.MsGraph", "MsGraph\Microsoft.Bot.Component.MsGraph.csproj", "{415A7327-9FA1-4354-8104-5900A8882582}"
```

Comment:
What about other components?

Reply:
Other .NET components will be added in when they're on main (orchestrator), the intention here is more that we can manage all of the SDK dependencies those would have here.